src/main/java/net/sacredlabyrinth/phaed/simpleclans/hooks/DiscordHook.java

```diff
@@ -480,4 +510,19 @@ private void updatePermissions(@NotNull ClanPlayer clanPlayer, @Nullable Clan cl
     enum DiscordAction {
         ADD, REMOVE
     }
+
+    protected static class LimitReachedException extends Exception {
+
+        public LimitReachedException() {
+            super();
+        }
+    }
+
+    protected static class InvalidChannelException extends Exception {
```

Comment:
We need a better name for this exception

Reply:
Or not. But passing the cause as a message would be good. Like new InvalidChannelMessage("channel already exists")